test/src/test/java/com/distkv/dst/test/core/operator/KVSListTest.java

```diff
@@ -1,13 +1,13 @@
-package org.dst.test.core.operator;
+package com.distkv.dst.test.core.operator;
 
 import java.util.ArrayList;
 import java.util.Arrays;
 import java.util.List;
 
```

Comment:
no blank line

Reply:
I renamed it with IntelliJ IDEA. It generated extra spaces for me. I have manually modified it again.